examples/search-replace/create-setters/README.md

```diff
@@ -0,0 +1,88 @@
+# search-replace: Create Setters Example
+
+The `search-replace` function can be used to search for fields using available matchers
+and add setter patterns as line comments in the resources.
+
+This is an end to end example depicting setter creation process using `search-replace` function.
+
+Let's start with the input resource
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-nginx
+spec:
+  replicas: 3
+  template:
+    spec:
+      containers:
+        - name: nginx
+          image: "gcr.io/nginx:1.14.2"
+```
+
+Suppose you want to expose the values of `replicas`, `image` and `tag` as parameters.
+You can create setters by invoking `search-replace` function with following arguments:
+
+```sh
+'by-path=spec.replicas' 'put-comment=kpt-set: ${replicas}'
```

Comment:
Shouldn't only the value be quoted?

Reply:
In general it is better to quote entire key=value as keys may also have regex characters.